.github/workflows/pr-labeler.yaml

```diff
@@ -0,0 +1,18 @@
+name: PR Labeler
+
+on:
+  workflow_call:
+    inputs:
+      configuration-path:
+        required: true
+        type: string
+
+jobs:
+  pr-labeler:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: TimonVS/pr-labeler-action@v3
+        with:
+          configuration-path: ${{inputs.configuration-path}}            
```

Comment:
This workflow needs a configuration file; how are we going to keep this file centrally updated? Should we add a step here to copy it from this repo when it is run remotely?

Reply:
For workflows, we are copying the whole `.github` directory when doing automatic update. That way, config files and scripts are also updated.

Reply:
Fair enough, but there aren't any config files here to copy, the scripts and the configs etc should also be here, right?

If you are pulling bellow the scripts that is fine but config files of workflows should be in this repo, think make it simple 👍